english/torch/torch.md

```diff
@@ -0,0 +1,4 @@
+torch.md
+
+Upon inspection the object has a small depression.
+When pressed an incandescent glow is emitted and the room comes into view.  [Return to the twisted passages](../zork/zork.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.